python/flatbuffers/builder.py

```diff
@@ -813,60 +843,40 @@ def Place(self, x, flags):
     """
 
     N.enforce_number(x, flags)
-    self.head = self.head - flags.bytewidth
-    encode.Write(flags.packer_type, self.Bytes, self.Head(), x)
+    new_head = self.head - flags.bytewidth
+    self.head = UOffsetTFlags.py_type(new_head)
+    encode.Write(flags.packer_type, self.Bytes, new_head, x)
 
   def PlaceVOffsetT(self, x):
     """PlaceVOffsetT prepends a VOffsetT to the Builder, without checking
 
     for space.
     """
     N.enforce_number(x, N.VOffsetTFlags)
-    self.head = self.head - N.VOffsetTFlags.bytewidth
-    encode.Write(packer.voffset, self.Bytes, self.Head(), x)
+    new_head = self.head - N.VOffsetTFlags.bytewidth
+    self.head = UOffsetTFlags.py_type(new_head)
+    encode.Write(packer.voffset, self.Bytes, new_head, x)
```

Comment:
where is `self.head = UOffsetTFlags.py_type(new_head)` coming from in these changes?  because i would expect self.Head() == self.head and Write itself did not change :confused: